workspace/apps/pidp/src/app/shared/components/success-dialog/success-dialog.component.spec.ts

```diff
@@ -1,12 +1,16 @@
-import { TestBed } from '@angular/core/testing';
```

Comment:
Add the following line to the top of the file to make the linter happy. For tests we don't mind using `any` :) 

`/* eslint-disable @typescript-eslint/no-explicit-any */`